database/sqlite/sqlite_aclk_chart.c

```diff
@@ -1003,8 +1004,12 @@ void aclk_update_retention(struct aclk_database_worker_config *wc, struct aclk_d
     if (!wc->host)
         hostname = get_hostname_by_node_id(wc->node_id);
 
-    log_access("ACLK STA [%s (%s)]: UPDATES %d RETENTION MESSAGE SENT. CHECKED %u DIMENSIONS.  %u DELETED, %u STOPPED COLLECTING",
-               wc->node_id, wc->host ? wc->host->hostname : hostname ? hostname : "N/A", wc->chart_updates, total_checked, total_deleted, total_stopped);
+    if (dimension_update_count < ACLK_MAX_DIMENSION_CLEANUP && !netdata_exit)
+        log_access("ACLK STA [%s (%s)]: UPDATES %d RETENTION MESSAGE SENT. CHECKED %u DIMENSIONS.  %u DELETED, %u STOPPED COLLECTING",
```

Comment:
Hello @stelfrag ,

Your PR is working as expected, but when I saw the logs there was something that called my attention:

```sh
/var/log/netdata/access.log:2022-05-31 15:31:03: ACLK STA [97b14fcd-0936-4336-87f4-b357bd8da106 (zeus)]: CHARTS ACKNOWLEDGED in the database upto 33349
/var/log/netdata/access.log:2022-05-31 15:32:13: ACLK STA [97b14fcd-0936-4336-87f4-b357bd8da106 (zeus)]: CHARTS ACKNOWLEDGED in the database upto 34522
/var/log/netdata/access.log:2022-05-31 15:32:28: ACLK STA [11149d59-91af-433b-96be-a239a91c58a4 (hades)]: Queued removed alerts.
/var/log/netdata/access.log:2022-05-31 15:32:28: ACLK STA [97b14fcd-0936-4336-87f4-b357bd8da106 (zeus)]: Queued removed alerts.
/var/log/netdata/access.log:2022-05-31 15:33:34: ACLK STA [87a9049b-477f-4a04-9561-0c6c0fcba546 (archlinux)]: UPDATES 1 RETENTION MESSAGE SENT. CHECKED 381 DIMENSIONS.  0 DELETED, 0 STOPPED COLLECTING
/var/log/netdata/access.log:2022-05-31 15:34:09: ACLK STA [11149d59-91af-433b-96be-a239a91c58a4 (hades)]: UPDATES 1 RETENTION MESSAGE SENT. CHECKED 6003 DIMENSIONS.  0 DELETED, 0 STOPPED COLLECTING
/var/log/netdata/access.log:2022-05-31 15:35:31: ACLK STA [97b14fcd-0936-4336-87f4-b357bd8da106 (zeus)]: UPDATES 1 RETENTION MESSAGE SENT. CHECKED 531 DIMENSIONS.  0 DELETED, 0 STOPPED COLLECTING
```

In `CHARTS ACKNOWLEDGED` we have only these words in uppercase, while the rest of the sentence is in lowercase. Is this difference really expected?

Another thing, I was talking with @DShreve2 , and the sentence `in the database upto` we missed a space between `up` and `to`. Can we also bring this simple fix for your PR?

Reply:
I adjusted the message. Also changed the "Queued removed alerts"  to uppercase